### ISSUE TITLE
Allow pointing sentry exceptions to self-hosted instances

### ISSUE THREAD

**Issue by dan-dr** [PostHog/posthog-js#175]
Hey guys.
LOVE posthog. We're using it for a new product.
We enabled sentry integrations, but every exception in posthog is hardcoded to go to the `sentry.io` domain, while we are self hosting it on our app's subdomain.

Happy to PR, would something like this work for you:
```javascript
// add prefix as last argument to be backwards compatible
PostHogLib.prototype.sentry_integration = function (_posthog, organization, projectId, prefix) { 
    // setupOnce gets called by Sentry when it intializes the plugin
    this.setupOnce = function (addGlobalEventProcessor) {
        addGlobalEventProcessor((event) => {
            if (event.level !== 'error' || !_posthog.__loaded) return event
            if (!event.tags) event.tags = {}
            event.tags['PostHog URL'] = _posthog.config.api_host + '/person/' + _posthog.get_distinct_id()
            let data = {
                $sentry_event_id: event.event_id,
                $sentry_exception: event.exception,
            }
            if (organization && projectId)
                data['$sentry_url'] =
                    (prefix || 'https://sentry.io/organizations/') + // prefix. for us it's https://errors.ourapp.com/ourapp/
                    organization +
                    '/issues/?project=' +
                    projectId +
                    '&query=' +
                    event.event_id
            _posthog.capture('$exception', data)
            return event
        })
    }
}

```

**Comment by macobo** [comment]
Sounds awesome, go ahead!

**Comment by dan-dr** [comment]
#176 :)

**Comment by macobo** [comment]
Released with 1.8.5
